Storage/StorageTests/Tools/StorageTypeDeletionsTests.swift

```diff
@@ -33,7 +33,7 @@ class StorageTypeDeletionsTests: XCTestCase {
         storage.deleteStaleAddOnGroups(siteID: sampleSiteID, activeGroupIDs: [123, 1234])
 
         // Then
-        let activeGroups = storage.loadAddOnGroups(siteID: sampleSiteID)
+        let activeGroups = storage.loadAddOnGroups(siteID: sampleSiteID).sorted { $0.groupID < $1.groupID }
```

Comment:
Thanks for catching this @ealeksandrov.

The `loadAddOnGroups` method is meant to sort items by name. I think that, by passing a name to  `createAddOnGroup` we will be reflecting a more real use-case for future readers. What do you think?